ui/packages/catalog-ui-search/src/main/webapp/js/DistanceUtils.js

```diff
@@ -73,4 +75,7 @@ module.exports = {
     // round the value, don't need picometer precision.
     return Math.round(value)
   },
+  coordinateRound: function(value) {
+    return parseFloat(parseFloat(value).toFixed(DECIMAL_PRECISION))
```

Comment:
❓ Do you need to wrap the final result in `parseFloat` again?

Reply:
Yup, `.toFixed()` returns a String.